app/component/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ReactNode } from "react";
+import * as styles from "./styles.css";
+
+export function Header(): ReactNode {
+  return (
+    <header className={styles.header}>
+      <div className={styles.headerMenu}>
+        <a href="/" className={styles.logo} />
```

Comment:
```suggestion
        <Link href="/" className={styles.logo} />
```
remixではaタグじゃなくてLinkが用意されているのでそちらを...
https://remix.run/docs/en/main/components/link#link